src/Service/PaymentMethodsFilterService.php

```diff
@@ -188,46 +188,6 @@ function (PaymentMethodEntity $paymentMethod) use ($brand, $adyenPluginId) {
         )->first();
     }
 
-    public function getAvailableGiftcards(
```

Comment:
Removing a public method is a breaking change. Could you please add `@deprecated` tag instead of removing this method?